GIBSDownloader/gibs_downloader.py

```diff
@@ -66,6 +82,16 @@ def remove_originals(originals_path, logging):
     shutil.rmtree(originals_path)
     os.mkdir(originals_path)
 
+def generate_video(originals_path, region, dates, video_path, xml_path, product):
+    if not os.path.isdir(video_path):
+        print("Generating video...", end="")
+        os.mkdir(video_path)
+        Animator.format_images(originals_path, region, dates, video_path, xml_path, product)
+        Animator.create_video(video_path)
+        print("done!")
```

Comment:
Might want to be more descriptive here with the output such as `Video generation has finished!`.